examples/ota-provider-app/esp32/main/main.cpp

```diff
@@ -138,7 +138,7 @@ static void InitServer(intptr_t context)
     }
     fclose(otaImageFile);
     otaImageFile = NULL;
-
+    chip::app::Clusters::OTAProvider::Init();
```

Comment:
This should be handled _by the cluster init function_.  The one in the cluster itself.  Consumers should not need to do this.

Reply:
So to be more clear:

1. Add this cluster to the list in `ClustersWithInitFunctions` in `src/app/common/templates/config-data.yaml`.
2. Implement a `MatterOTAProviderPluginServerInitCallback` function in the cluster .cpp that does the init.
3. Check that it's actually being called.

Reply:
In particular, the main question is whether `USE_ZAP_CONFIG` is set to `1` in the cases we care about in `src/app/util/DataModelHandler.cpp`

Reply:
And if not (e.g. because all this stuff uses `EmptyDataModelHandler.cpp` instead of `DataModelHandler.cp`, then we should still call the function the right thing so that it matches all the other init functions, and then think about where the calls to it should actually go.

Reply:
resolved.